wrappers/outputs.tf

```diff
@@ -1,5 +1,5 @@
 output "wrapper" {
   description = "Map of outputs of a wrapper."
   value       = module.wrapper
-  # sensitive = false # No sensitive module output found
+  sensitive   = true # Set to true to hide the sensitive output in the console
```

Comment:
You should not specify `sensitive = true` manually here because the main module does not have sensitive [outputs](https://github.com/terraform-aws-modules/terraform-aws-cloudfront/blob/master/outputs.tf).

Terraform AWS provider does not mark such outputs sensitive in the first place, so we can't do anything here to override it. This is the standard behaviour.